src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -0,0 +1,264 @@
+                getAngle().getDegrees() - encoderOffset,
+                SwerveConstants.ANGLE_GEAR_RATIO);
+        // Set the integrated angle encoder to the absolute position.
+        CTREUtil.checkError(
+                () -> mAngleMotor.setSelectedSensorPosition((int) absolutePosition, 0,
+                        50), 3);
+    }
+
+    /**
+     * Configures the angle encoder to the given constants.
+     */
+    private void configAngleEncoder() {
+        CTREUtil.checkError(() -> angleEncoder.configFactoryDefault(30), 3);
+        CTREUtil.checkError(
+                () -> angleEncoder.configFeedbackNotContinuous(true, 30), 3);
+        CTREUtil.checkError(
+                () -> angleEncoder.configSelectedFeedbackSensor(
+                        TalonSRXFeedbackDevice.CTRE_MagEncoder_Absolute, 0, 30),
+                3);
+        angleEncoder.setSensorPhase(SwerveConstants.ENCODER_INVERT);
+    }
+
+    /**
+     * Configures the angle motor to the given constants.
+     */
+    private void configAngleMotor() {
+        CTREUtil.checkError(() -> mAngleMotor.configFactoryDefault(30), 3);
+        CTREUtil.checkError(
+                () -> mAngleMotor.configAllSettings(
+                        SwerveConstants.ANGLE_OPEN_LOOP_RAMP,
+                        SwerveConstants.ANGLE_CLOSED_LOOP_RAMP,
+                        constants.anglePIDCoefs, 30),
+                3);
+        CTREUtil.checkError(() -> mAngleMotor.configSupplyCurrentLimit(
+                SwerveConstants.ANGLE_CURRENT_LIMIT, 30), 3);
+        mAngleMotor.setInverted(SwerveConstants.ANGLE_MOTOR_INVERT);
+        mAngleMotor.setNeutralMode(SwerveConstants.ANGLE_NEUTRAL_MODE);
+        mAngleMotor.setSensorPhase(SwerveConstants.ENCODER_INVERT);
+        CTREUtil.checkError(() -> mAngleMotor.configRemoteFeedbackFilter(
+                angleEncoder.getDeviceID(),
+                RemoteSensorSource.TalonSRX_SelectedSensor, 0, 30), 3);
+        CTREUtil.checkError(() -> mAngleMotor.configSelectedFeedbackSensor(
+                FeedbackDevice.IntegratedSensor, 0, 30), 3);
+        resetToAbsolute();
+    }
+
+    /**
+     * Configures the drive motor to the given constants.
+     */
+    private void configDriveMotor() {
+        CTREUtil.checkError(() -> mDriveMotor.configFactoryDefault(30), 3);
+        CTREUtil.checkError(
+                () -> mDriveMotor.configAllSettings(
+                        SwerveConstants.DRIVE_OPEN_LOOP_RAMP,
+                        SwerveConstants.DRIVE_CLOSED_LOOP_RAMP,
+                        constants.drivePIDCoefs, 30),
+                3);
+        CTREUtil.checkError(() -> mDriveMotor.configSupplyCurrentLimit(
+                SwerveConstants.DRIVE_CURRENT_LIMIT, 30), 3);
+        mDriveMotor.setInverted(SwerveConstants.DRIVE_MOTOR_INVERT);
+        mDriveMotor.setNeutralMode(SwerveConstants.DRIVE_NEUTRAL_MODE);
+        CTREUtil.checkError(() -> mDriveMotor.setSelectedSensorPosition(0, 0, 30), 3);
+    }
```

Comment:
A. magic numbers, add variables at top of function
B. MOTOR CONFIG and TrigonTalonFx